app/views/shared/sidebar/_sidebar.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class='sidebar'>
+  <div class='text-white w-3/4 self-center my-10'>
+    <%= inline_svg_tag 'logo.svg' %>
+  </div>
+  <% sidebar_sections.each do |section| %>
+    <%= render 'shared/sidebar/sidebar_item', name: section[:name], path: section[:path], sections: sidebar_sections %>
```

Comment:
is it necessary to pass `sections: sidebar_sections` param?

Reply:
nope, it wasn't, fixed!